Add text for the Devpost submission site (#19)
pivlab/manugen-ai: packages/manugen-ai/docs/devpost/additional_info.md

# Upload a file

I've added the agent architecture diagram.

# Testing Instructions for application

First, we encourage you to watch our short video about Manugen-AI here: 1. TODO: add video link

To test Manugen-AI, you can either use our hosted demo or test it locally in your computer.

> ## Demo

Reviewer: ```suggestion
## Hosted demo
```

1. Go to: https://manugen-ai.cu-dbmi.dev/
2. Click on the lightbulb to load the Markdown example we used in the video.
3. To draft the manuscript, you have to go one section at a time (Introduction, Results, etc) by selecting the entire section content and click on the "Draft" action, and wait for the text to come back.

> 4. You can also edit an already drafted section by adding instructions in the middle of the text, selecting the entire section, and clicking on "Draft".

Reviewer: ```suggestion
4. Drafted sections may be further enhanced by adding instructions to the text, selecting the entire section, and clicking on "Draft".
```

>    The system will edit the existing draft by following your instructions.

Reviewer: ```suggestion
   The system will edit the existing draft and follow your additional instructions.
```

> 5. You can also upload figures (it works with PNG format) by clicking on the "Toggle Attachments" button at the top left. If you want to download figures for the example, you can get them from [here](https://github.com/pivlab/manugen-ai/tree/main/frontend/public/example).

Reviewer: ```suggestion
5. Figures may be uploaded (it works with PNG format) by clicking on the "Toggle Attachments" button at the top left. If you want to download figures for the example, please see [here](https://github.com/pivlab/manugen-ai/tree/main/frontend/public/example).
```

6. Once a figure is uploaded, you'll see it gets a figure number like "Figure 1", a title and a description. You can then reference this figure using its figure number (like "Figure 1") in the Results section, and the system will use the Figure title and description.

> ## Local test

Reviewer: ```suggestion
## Local demo
```

1. Clone our GitHub repository: https://github.com/pivlab/manugen-ai
2. Follow the instructions in the `README.md`, under section [Installation](https://github.com/pivlab/manugen-ai#installation).

> 3. Once it's ready, you'll use the URL link to access the frontend locally (such as http://localhost:8900/) and start testing the system.

Reviewer: ```suggestion
3. Once it's ready, you'll use the URL link to access the frontend locally (such as http://localhost:8900/) and start interacting with the system.
```

The steps to test the system are the same as described in the "Demo" section above.

# BONUS (Optional)

## A link to your published blog post, video, or podcast

> TODO: add link to video

Reviewer: Will add this in a forthcoming update with a link to the Pividori Lab blog.
